pages/dkp/kommander/2.2/networking/use-istio/index.md

```diff
@@ -0,0 +1,116 @@
+title: Integrate microservices using Istio
+menuWeight: 24
+excerpt: Learn how to integrate microservices managed by Istio into a DKP cluster
+beta: false
+experimental: true
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+Istio helps you manage cloud-based deployments by providing an open-source service mesh to connect, secure, control, and observe microservices.
+
+This tutorial demonstrates how to expose an application running on the DKP cluster using the LoadBalancer (layer-4) service type.
+
+<p class="message--note"><strong>NOTE: </strong>Usage and installation of Istio on DKP is currently a self-service feature.</p>
+
+## Before you begin
+Before starting this tutorial, you should verify the following:
+
+- You must have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You must have a properly deployed and running cluster. For information about deploying Kubernetes with default settings, see the [Quick start][quickstart].
+
+- You must have `cert-manager` enabled and deployed through your `cluster.yaml` addon configuration.
+
+## Deploy Istio using DKP
+
+Review the [list of available applications](../../platform-applications#workspace-platform-applications) that can be enabled to deploy to your attached cluster.
+
+1.  Enable a supported application to deploy to [your existing attached cluster](../../../../clusters/attach-cluster/) with an `AppDeployment` resource.
+
+1.  Within the `AppDeployment`, define the `appRef` to specify which `App` will be enabled:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: apps.kommander.d2iq.io/v1alpha2
+    kind: AppDeployment
+    metadata:
+      name: istio
+      namespace: ${WORKSPACE_NAMESPACE}
+    spec:
+      appRef:
+        name: istio-1.11.6
+        kind: ClusterApp
+    EOF
+    ```
+
+1.  Create the resource in the workspace you just created, which instructs Kommander to deploy the `AppDeployment` to the `KommanderCluster`s in the same workspace.
+
+<p class="message--note"><strong>NOTE: </strong>The <code>appRef.name</code> must match the app <code>name</code> from the list of available applications.</p>
+
+## Download the Istio command line utility
+
+1. Store a local environment variable containing the current Istio version running in your cluster:
+
+    ```bash
+    export KONVOY_ISTIO_VERSION="$(kubectl get clusteraddons istio -o go-template='{{ .spec.chartReference.version }}')"
+    ```
+
+2. Pull a copy of the corresponding Istio command line to your system:
+
+    ```bash
+    curl -L https://istio.io/downloadIstio | ISTIO_VERSION=${KONVOY_ISTIO_VERSION} sh -
+    ```
+
+3. Change to the Istio directory and set your PATH environment variable by running the following commands:
+
+    ```bash
+    cd istio*
+    export PATH=$PWD/bin:$PATH
+    ```
+
+4. You should now be able to run the following commands and see output like this:
```

Comment:
Are we missing the output? Or is the following a mix of command and output? In that case, maybe we could separate?

Reply:
Yes. Right now this example is a mix, it should be broken up into a bash section with just the 'istioctl version' line (no `$`!) and then a 'sh' section for the stuff after.